assets/dev/scss/frontend/general/_button.scss

```diff
@@ -83,19 +91,32 @@
 
 	span {
 		text-decoration: inherit; //fix for conflict with text-decoration & inline-block
+		align-self: center;
+	}
+
+	&:hover {
+		.elementor-button-icon {
+			i {
+				color: var(--button-icon-color-hover);
+				transition: all 0.3s;
+			}
+		
+			svg {
+				fill: var(--button-icon-color-hover);
+				transition: all 0.3s;
+			}
+		}
 	}
 }
 
 .elementor-element.elementor-button- {
 	@include button-types( '.elementor-button' );
-
 }
 
 .elementor-widget-button {
 	.elementor-button {
 		.elementor-button- {
-
 			@include button-types( '' );
 		}
 	}
-}
```

Comment:
Please restore the original version. I think we need to remove one line at the end.

Reply:
I removed the line at the end